shark/examples/shark_eager/dynamo_demo.ipynb

```diff
@@ -36,7 +36,9 @@
     "    from torchdynamo.optimizations.backends import create_backend\n",
     "    from torchdynamo.optimizations.subgraph import SubGraph\n",
     "except ModuleNotFoundError:\n",
-    "    print(\"Please install TorchDynamo using pip install git+https://github.com/pytorch/torchdynamo\")\n",
```

Comment:
Why are these changes included in this PR? Is this coming from the linter?

Reply:
yeah. Each time I run formatter, it pops up. So I thought to put it up. Just the formatting thing.

Reply:
interesting, I don't get that. Let me see if my formatter tries to put it back.

Reply:
oh it's probably because you have this installed
```
pip install black[jupyter]
```
and no one else does. We can just leave this in then if it's annoying to take out.

Reply:
yes, black[jupyter] is there in my env.